### PR TITLE
UI integration tests

### PR DESCRIPTION
**Pull request by CameronKroeker47**
These are all of my tests for the UI implimentation. 
These changes do not include the Package-lock.json file or package.json file. Let me know if I should include those files as well.

### CODE REVIEW

File: src/line/line.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h1>Line Chart Builder</h1>
                     </div>
                     <div class="content">
                         <div class="data-entry">
-                            <div id="x-y-data-grid">
+                            <div id="x-y-data-grid" data-testid = "coordvalues">
```
Comment (piedrabryan) [inline src/line/line.html:35]: I might be blind, but I couldn't find where you were using the data-testid?

File: src/graphApp.test.js [hunk 2]
```diff
@@ -0,0 +1,325 @@
+/**
```
Comment (piedrabryan) [inline src/graphApp.test.js:1]: Really good tests, the only recommendation I would have is fixing the weird indentations and possibly more comments, but those aren't issues, more for convenience.

File: src/line/line.html (hunk 1 above)
Comment (CameronKroeker47) [reply src/line/line.html:35]: I compleltly forgot I had added that. I found a way to reference that element without the test id. The data-testid = "coordvalues" can be removed. It isn't nessesary.

**Review by jade-fleskes — COMMENTED** [review]
this looks good to me. The only thing that stands out at all is inconsistent indentation, but that's purely aesthetic

File: src/graphApp.test.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    expect(generateChartImg).toHaveBeenCalledWith
+    (
+        "line",
+         [{"x": "1", "y": "2"},{"x": "3", "y": "4"},{"x": "5", "y": "6"}], 
+          "X axis title",
+          "Y axis title",
+          "Test title",
+          "#00ff4c"
+    )
+
```
Comment (BillquistErik) [inline src/graphApp.test.js:324]: All of the test look good and are very well written. Only thing i'd change is what brian mentioned

**Review by BillquistErik — COMMENTED** [review]
good work

**Pull request closed without merging**